nx-X11/programs/Xserver/GL/mesa/Imakefile.inc

```diff
@@ -1,9 +1,5 @@
 XCOMM $XFree86: xc/lib/GL/mesa/src/Imakefile.inc,v 1.7tsi Exp $
 
-#ifndef MesaBuildDir
-#define MesaBuildDir $(GLXLIBSRC)/mesa/
-#endif
-
 MESABUILDDIR = MesaBuildDir
```

Comment:
On  So 14 Jun 2015 01:50:17 CEST, Mihai Moldovan wrote:

>> @@ -1,9 +1,5 @@
>>  XCOMM $XFree86: xc/lib/GL/mesa/src/Imakefile.inc,v 1.7tsi Exp $
>>
>> -#ifndef MesaBuildDir
>> -#define MesaBuildDir $(GLXLIBSRC)/mesa/
>> -#endif
>> -
>>  MESABUILDDIR = MesaBuildDir
>
> Hm, no, this won't work.
>
> `MESABUILDDIR` will be empty, but seems to be used extensively in  
> `nx-X11/lib/GL/mesa/main/Imakefile.inc`, which I guess is included  
> by all mesa "subprojects"?

It seems that compiler steps are referenced twice.

1) nx-X11/programs/Xserver/GL/mesa/GLcore/Imakefile

-> Mesa*BuildDir(s) get set to the same dirs as the .c files that are  
about to be built.

2) nx-X11/programs/Xserver/GL/mesa/*/Imakefile

-> Here Mesa*BuildDir variables are (I guess) not defined, resuling in  
building the
    .o files in the same dir as the .c files.

Both approaches are intended.

> I'm not sure what to do about this, though, short of resurrecting  
> `GLXLIBSRC`.

Nope (IMHO). See above.

-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/freebusy/m.gabriel%40das-netzwerkteam.de.xfb